src/LinuxTracing/CMakeLists.txt

```diff
@@ -96,3 +96,12 @@ target_link_libraries(LinuxTracingTests PRIVATE
         GTest::Main)
 
 register_test(LinuxTracingTests)
+
+add_executable(SampleProgram)
```

Comment:
I think `SampleProgram` is too generic. What `ThreadDependencyTestTarget` or `ThreadDepedencyTestApp` or something similar?